app/controllers/v0/education_enrollments_controller.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+module V0
+  class EducationEnrollmentsController < ApplicationController
+    def show
+      # TODO: fetch for realz
+      ee = FactoryGirl.build(:education_enrollment)
```

Comment:
Please do not merge until we address this line.  We either need to comment it out or wait until we are able to fetch real data from EVSS.